assembler/manual.html

```diff
@@ -706,13 +708,13 @@ <h4>Input data</h4>
 
 <p>
     <code>--pacbio &lt;file_name> </code><br>
-    &nbsp;&nbsp;&nbsp;&nbsp;File with PacBio CLR reads. For PacBio CCS reads use <code>-s</code> option. More information on PacBio reads is provided in <a href="#pacbio">section 3.1</a>. 
+    &nbsp;&nbsp;&nbsp;&nbsp;File with PacBio CLR reads for late scaffolding step. For PacBio CCS reads use <code>-s</code> option. More information on PacBio reads is provided in <a href="#pacbio">section 3.1</a>.
```

Comment:
The change is not quite correct. There is no "late scaffolding" step in SPAdes at all.

Reply:
Then please update the manual accordingly. Elsewhere you state nanopore and pacbio reads are used only for scaffolding.

Reply:
The section 3.1 of manual says, that "... SPAdes will use PacBio CLR and Oxford Nanopore reads for gap closure and repeat resolution.". This correctly represents the behavior.